src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+import duke.parser.Comd;
+
+/**
+ * A class for storing information for commands.
+ */
+public class Command {
+    private Comd comd;
+    private String input;
+    private String output;
+    private ArrayList<Task> taskList;
+
+    public Command() {
+    }
+
+    public Command(Comd comd, String input) {
```

Comment:
How about using subclassing to represent the different kind of Commands? You could capitalise on polymorphism.